ember-test-selectors/README.md

```diff
@@ -23,9 +23,11 @@ More information on why that is useful are available on our
 Compatibility
 ------------------------------------------------------------------------------
 
-- Ember 3.8 or above
+- Ember 3.8 to Ember 4.12
```

Comment:
so we don't support Ember 5 with this plugin? 😂 is that real? or is it just that we're not testing for it?

Reply:
The accurate answer is: 
- It was not tested
- I added the scenario to see what it does
- 5+ failed
- I removed the scenarios and updated the README

I think that's a job for a separate PR ^^